modules/react/table/lib/TableRow.tsx

```diff
@@ -192,16 +192,22 @@ const Row = styled('tr')<TableRowProps>(
   }
 );
 
-export default class TableRow extends React.Component<TableRowProps> {
-  public static State = TableRowState;
-
-  public render() {
-    const {state, header, children, ...elemProps} = this.props;
-
+const TableRow = createComponent('tr')({
+  displayName: 'TableRow',
+  Component: (
+    {state, header, children, ...elemProps}: TableRowProps & StyledType,
```

Comment:
This is a bit weird. The external interface already supports the `as` prop, so `StyledType` shouldn't be needed here. It would be needed on the `styled` function if it wasn't a string.